### ISSUE TITLE
self.model_config['config'] is a list in model

### ISSUE THREAD

**Issue by CescMessi** [MarsTechHAN/keras2ncnn#39]
I want to convert [this model](https://github.com/UPCCV/Face-Attractiveness/blob/master/web/model/model.h5) to ncnn but failed. I found that the `self.model_config['config']` in this model is a list, which caused error.

```
Traceback (most recent call last):
  File "/home/tommy/miniconda3/envs/pytorch/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/tommy/miniconda3/envs/pytorch/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/tommy/miniconda3/envs/pytorch/lib/python3.8/site-packages/keras2ncnn/__main__.py", line 12, in <module>
    sys.exit(_main())
  File "/home/tommy/miniconda3/envs/pytorch/lib/python3.8/site-packages/keras2ncnn/keras2ncnn.py", line 55, in main
    H5dfParser(args.input_file).parse_graph(keras_graph)
  File "/home/tommy/miniconda3/envs/pytorch/lib/python3.8/site-packages/keras2ncnn/h5df_parser.py", line 111, in parse_graph
    for layers in self.model_config['config']['layers']:
TypeError: list indices must be integers or slices, not str
```

**Comment by MarsTechHAN** [comment]
I think this is probably using some very old Keras format. Fixed in 96d7f1b. You can take a try if it gives out correct result.

**Comment by CescMessi** [comment]
> I think this is probably using some very old Keras format. Fixed in [96d7f1b](https://github.com/MarsTechHAN/keras2ncnn/commit/96d7f1b95a7bc29236f9021c9c2b92fb0eb026ec). You can take a try if it gives out correct result.

It works well, thank you.
